config/routes.rb

```diff
@@ -0,0 +1,58 @@
+Rails.application.routes.draw do
+  get 'to_do_lists/index'
+
```

Comment:
@vlasiak, FYI: I'm expecting that soon that will be changed into resources (while working on next user stories).